### PR TITLE
Feature/convert wait statements issue#83

### PR DESCRIPTION
**Pull request by grasbergerm**

### CODE REVIEW

**Comment by coveros-ci-dev** [comment]
SonarQube analysis reported 4 issues
* ![MAJOR][MAJOR] 3 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [WaitFor.java#L44](https://github.com/Coveros/selenified/blob/080579233c5561a54c7e504a32225c126203022c/src/main/java/com/coveros/selenified/application/WaitFor.java#L44): Remove this unused "log" private field. [rule](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1068)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/main/java/com/coveros/selenified/application/WaitFor.java [hunk 1]
```diff
@@ -22,8 +22,10 @@
 
 import com.coveros.selenified.OutputFile;
 import com.coveros.selenified.OutputFile.Result;
-import org.openqa.selenium.NoAlertPresentException;
+import org.openqa.selenium.TimeoutException;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
 import org.testng.log4testng.Logger;
```
Comment (coveros-ci-dev) [inline src/main/java/com/coveros/selenified/application/WaitFor.java]: MINOR Remove this unused import 'org.testng.log4testng.Logger'. [rule](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by coveros-ci-dev** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/main/java/com/coveros/selenified/element/Element.java [hunk 2]
```diff
@@ -390,7 +390,7 @@ public Equals assertEquals() {
      * @throws InvalidLocatorTypeException: if a bad locator type is passed, an invalid locator type exception will
      *                                      be thrown
      */
-    private By defineByElement() throws InvalidLocatorTypeException {
+        By defineByElement() throws InvalidLocatorTypeException {
```
Comment (msaperst) [inline src/main/java/com/coveros/selenified/element/Element.java]: spacing is all messed up here

**Pull request merged**